favorite_files.py

```diff
@@ -28,8 +28,23 @@ def run(self):
 class SelectFavoriteFileCommand(sublime_plugin.WindowCommand):
     """Open the selected favorite(s)."""
 
+    def prompt_for_alias(self, index, name, group_name=None):
+        """
+        Prompt for an alias for the favorite file.
+
+            index      : the evaluated quick panel index
+            name       : a tuple (alias, path)
+            group_name : the group browsed in the quick panel
+        """
+
+        w = sublime.active_window()
+        # sending the variables to Favs, needed there for set_alias()
+        Favs.changing_alias_for = index, name, group_name
+        w.show_input_panel("Alias for this file:", name[0],
+                           Favs.set_alias, None, None)
+
     def open_file(self, value, group=False):
-        """Open the file(s)."""
+        """Open the file(s). Also called when just editing aliases."""
```

Comment:
Not sure we should be using the open file to edit alias.  I'll try and pull this down locally and take a better look at the approach.  I'm sure some of this could be cleaned up, but I want to play with it before I make any suggestions.